src/Bundle/CoverallsBundle/Command/CoverallsJobsCommand.php

```diff
@@ -168,13 +174,19 @@ protected function loadConfiguration(InputInterface $input, $rootDir)
     /**
      * Execute Jobs API.
      *
-     * @param Configuration $config configuration
+     * @param InputInterface $input  input arguments
+     * @param Configuration  $config configuration
      *
      * @return bool
      */
-    protected function executeApi(Configuration $config)
+    protected function executeApi(InputInterface $input, Configuration $config)
```

Comment:
this looks like a BC breaker, as this class is not marked with an internal. can you think about way to bring this feature without BC break ?

Reply:
Yes, sure.

Reply:
Could you check this solution, please.